BackendApartmentReservation/BackendApartmentReservation.sln

```diff
@@ -3,7 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15

```

Comment:
Is this file needed in this PR? Seems like something that should have been ignored.  Might be wrong.

Reply:
hmm thats a good question. I think .sln file is the main file, combining all the projects. Its what you open in VS